README.md

```diff
@@ -12,7 +12,9 @@ Welcome! This repository is the “standard library” for [Semgrep](https://sem
 
 ## How do I use these rules?
 
-We recommend starting with a pre-selected subset of rules that someone has already carefully tuned for precision. These subsets of rules, which we will call rulesets, may include rules not present in this repository. However, they are available for free on the [Semgrep Registry](https://semgrep.dev/explore).
+We recommend running Semgrep with the command `semgrep --config auto`. The `--config auto` flag will automatically fetch rules relevant to your project from [Semgrep Registry](https://semgrep.dev/explore). Only the name of your project will be sent to Semgrep Registry, to look up the project's language and select relevant rules. Your source code will not be uploaded.
```

Comment:
I specified that 
1. their project name is sent to Semgrep Registry (is this actually where it's sent, or should I say something more generic?)
2. The only reason why is to look up the project's language makeup and select relevant rules. Please let me know if this is correct :)